### PR TITLE
Added ASL postures

### PR DESCRIPTION
**Pull request by Vittorio-Caggiano**
update to register additional posture for the ASL task

### CODE REVIEW

**Review by vikashplus — COMMENTED** [review]
Do we still need this PR? 
- In our last discussion on the topic we decided to just to training on random poses and eval on ASL poses. Are we still aligned on it?
- If yes, should we cancel this PR?

File: mj_envs/envs/biomechanics/__init__.py [hunk 1]
```diff
@@ -142,46 +142,55 @@
...
+                'viz_site_targets': ('THtip','IFtip','MFtip','RFtip','LFtip'),
+                'target_jnt_value': np.array(ASL_qpos[k],'float'),
+                'normalize_act': False,
+                'reset_type': "none",        # none, init, random
+                'target_type': 'fixed',      # switch / generate/ fixed
+            }
+    )
+
+## create one environments where the posture is chosen randomly from 0 to 9
+m = np.array([ASL_qpos[i] for i in range(10)])
```
Comment (Vittorio-Caggiano) [inline mj_envs/envs/biomechanics/__init__.py:176]:
```suggestion
m = np.array([ASL_qpos[i] for i in range(10)]).astype(float)
```

File: mj_envs/envs/biomechanics/__init__.py [hunk 2]
```diff
@@ -142,46 +142,55 @@
...
+Rpos = {}
+for i_n, n  in enumerate(jnt_namesHand):
+    Rpos[n]=m[:,i_n].astype(float)
```
Comment (Vittorio-Caggiano) [inline mj_envs/envs/biomechanics/__init__.py:179]:
```suggestion
    Rpos[n]=(np.min(m[:,i_n]), np.max(m[:,i_n]))
```

**Review by Vittorio-Caggiano — COMMENTED** [review]
This PR is still valid with the changes I suggested to better represent the range of random poses. 
In addition, l we need the generation of single environments for single ASL for testing.

**Comment by vikashplus** [comment]
Sounds good, Let's merge it in. Seems like there are merge conflicts. Can you attend to them and post outputs of following to this conversation.
- python tests/test_envs.py
- pytest tests/envs/

**Comment by Vittorio-Caggiano** [comment]
@vikashplus sorry, I missed this PR... fixed the conflicts and they are ready now to merge

**Review by vikashplus — APPROVED** [review]

**Comment by vikashplus** [comment]
Please post an output of all tests passing before you merge.

**Comment by vikashplus** [comment]
@Vittorio-Caggiano 
`dambrl` is stale now. If this PR is still required please push it to `V1`. I'll close this PR. Feel free to create a new one.

**Pull request closed without merging**